wcet/Makefile

```diff
@@ -0,0 +1,342 @@
+# Makefile to build and run the examples using KLEE.
```

Comment:
@rasoolmaghareh This is fine for now, but this new directory should be a part of a separate PR. This Makefile contains parts that are useful only for scalability directory, such as the tr target, and its DEFAULT_EXPERIMENT_SET, for example, does not match programs in the directory. Would you please remove parts of scalability folder that are irrelevant for this new directory.

Please also don't forget to add an entry in the README.md on this new folder.